client/trino-client/src/main/java/io/trino/client/StatementClientV1.java

```diff
@@ -80,6 +80,7 @@
     private final AtomicReference<String> setCatalog = new AtomicReference<>();
     private final AtomicReference<String> setSchema = new AtomicReference<>();
     private final AtomicReference<String> setPath = new AtomicReference<>();
+    private final AtomicReference<String> setRoutingGroup = new AtomicReference<>();
```

Comment:
This field isn't needed. These `set` fields are used to handle responses from the server that the client needs to keep in its local session state.